CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewPane.java

```diff
@@ -941,6 +944,25 @@ private void loadPreferences() {
             }
         }
     }
+    
+    private Node createPage(final int pageIndex, final List<ObservableList<String>> rows) {
+        if (rows != null) {
+            final int fromIndex = pageIndex * MAX_ROWS_PER_PAGE;
+            final int toIndex = Math.min(fromIndex + MAX_ROWS_PER_PAGE, rows.size());
+
+            table.setItems(FXCollections.observableArrayList(rows.subList(fromIndex, toIndex)));
+        }
+        
+        return new BorderPane(table);
+    }
+    
+    private void paginate(final List<ObservableList<String>> rows) {
+        final Pagination pagination = new Pagination(table.getItems().size() / MAX_ROWS_PER_PAGE + 1);
```

Comment:
Have you checked whether sorting and filtering still work across all table data?

Reply:
Haven't gotten to that yet. Like I said previously, there is still a bit of testing to be done and this would be part of it